pint_pandas/testsuite/test_issues.py

```diff
@@ -41,7 +42,7 @@ def test_force_ndarray_like(self):
             expected = pd.DataFrame(
                 {0: PintArray(q_a_), 1: PintArray(q_b)}, dtype="pint[degC]"
             )
-            self.assert_equal(result, expected)
+            tm.assert_equal(result, expected)
```

Comment:
what's caused the change to use tm instead of self?

Reply:
When I dragged in `test_map` from the pandas test cases, it used `tm.assert_series_equal` instead of `self.assert_series_equal`.  I changed everything to use `tm.` under one theory of consistency, but now realize that's exactly backwards.  An EA should use self methods, not static methods from base classes.  I'll revert those.

Reply:
OK that's fine for now, we can change it a later PR if needed.

it looks like this change to pandas is the cause https://github.com/pandas-dev/pandas/pull/54355
interestingly geopandas uses `from pandas.testing import assert_series_equal`, and there's frame, index and EA functions